### PR TITLE
Test for previewing title changes through preview URL

### PR DESCRIPTION
**Pull request by igladun**
Simplified version of changes preview test

### CODE REVIEW

**Comment by igladun** [comment]
@hoverduck @alisterscott, sorry, forget to set you as reviewers.

File: lib/base-container.js [hunk 1]
```diff
@@ -94,4 +94,8 @@ export default class BaseContainer {
 
 		return this.waitForPage();
 	}
+
+	navigateBack() {
```
Comment (hoverduck) [inline lib/base-container.js]: I think this would make more sense to go in `driver-helper` rather than `base-container`

**Comment by alisterscott** [comment]
I've had to revert this in #942 because for some reason it doesn't work when running against calypso.localhost:3000, the back button remains on the front end of the site and the customizer step then fails.

@igladun: would you mind having a look at why this is the case on calypso.localhost:3000?

**Comment by igladun** [comment]
@alisterscott , I will check

**Pull request merged**